test/e2e/presto_prometheus_connector_up_test.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"bytes"
+	"github.com/operator-framework/operator-metering/test/reportingframework"
+	"github.com/stretchr/testify/require"
+	"os"
+	"os/exec"
+	"strings"
+
+	"testing"
+)
+
+func testPrometheusConnectorWorks(t *testing.T, rf *reportingframework.ReportingFramework) {
+	t.Helper()
+	out, err := exec.Command(
+		"/usr/local/bin/kubectl",
+		"-n", rf.Namespace,
+		"get", "pods",
+		"-l", "app=presto",
```

Comment:
Hmm while we know that this will always return the `presto-coordinator` pod as we know the context of the manual install, but there are no guarantees that there are no presto-workers pods.

I think we're more looking for:
```go
"-l", "app=presto,presto=coordinator",
```

This way, we're guaranteed to get back the presto coordinator pod in the current namespace.

Reply:
ah, okay. done.